frontend/src/routes/Infrastructure/Clusters/ManagedClusters/ManagedClusters.test.tsx

```diff
@@ -515,11 +516,12 @@ describe('Clusters Page', () => {
 
 describe('Clusters Page RBAC', () => {
     test('should perform RBAC checks', async () => {
-        const rbacCreateManagedClusterNock = nockRBAC(rbacCreate(ManagedClusterDefinition))
+        nockIgnoreApiPaths()
+        const rbacCreateManagedClusterNock = nockRBAC(rbacCreateTestHelper(ManagedClusterDefinition))
         const upgradeRBACNocks: Scope[] = upgradeableMockManagedClusters.reduce((prev, mockManagedCluster) => {
             prev.push(
-                nockRBAC(getClusterCuratorPatchResourceAttributes(mockManagedCluster.metadata.name!)),
-                nockRBAC(getClusterCuratorCreateResourceAttributes(mockManagedCluster.metadata.name!))
+                nockRBAC(getClusterCuratorPatchResourceAttributes(mockManagedCluster.metadata.name!)).persist(),
+                nockRBAC(getClusterCuratorCreateResourceAttributes(mockManagedCluster.metadata.name!)).persist()
```

Comment:
Did we affect the number of times these RBAC checks are performed?

Reply:
I think a bug that has since been corrected was affecting the frequency of calls...removing the ".persist()" does not appear to change the result of the test.